.github/workflows/build.yaml

```diff
@@ -0,0 +1,76 @@
+name: Build backend
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - 'v*'
+    paths:
+      - 'expense-backend/**'
+  pull_request:
+    branches:
+      - "**"
+    paths:
+      - 'expense-backend/**'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}-backend
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./expense-backend
+
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v3
+      - name: Use Node.js 20
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: 'npm'
+          cache-dependency-path: './expense-frontend/package-lock.json'
```

Comment:
This step is going to executed in the working-directory `./expense-backend`. How could we resolve `cache-dependency-path: './expense-frontend/package-lock.json'` there? It doesn't make sense. It's not also making sense to use the cache key provided by the frontend for something backend-related